service-workers/service-worker/request-end-to-end.https.html

```diff
@@ -23,13 +23,12 @@
           assert_equals(result.method, 'GET', 'request.method');
           assert_equals(result.referrer, location.href, 'request.referrer');
           assert_equals(result.mode, 'navigate', 'request.mode');
-          assert_equals(result.request_construct_error, 'TypeError',
+          assert_equals(result.request_construct_error, '',
                         'Constructing a Request with a Request whose mode ' +
-                        'is navigate and non-empty RequestInit must throw a ' +
-                        'TypeError.')
+                        'is navigate and non-empty RequestInit should not throw');
           assert_equals(result.credentials, 'include', 'request.credentials');
           assert_equals(result.redirect, 'manual', 'request.redirect');
-          assert_equals(result.headers['user-agent'], undefined,
+          assert_true(result.headers['user-agent'] === undefined,
```

Comment:
@youennf Why this change from `assert_equals(t, undefined)` to `assert_true(t === undefined)`?